package.json

```diff
@@ -577,6 +578,7 @@
         "@vscode/test-electron": "^2.1.5",
         "eslint": "^7.25.0",
         "eslint-plugin-import": "^2.22.1",
+        "fs-extra": "^11.1.0",
```

Comment:
Do we actually need `fs-extra` (even as a dev dependency)?

Reply:
This is just for the gulpfile, so we technically don't need it. But we can't use the vscode `fs`  to replace it.